.github/workflows/master.yml

```
@@ -28,7 +28,28 @@ jobs:
     needs: build
     runs-on: ubuntu-latest
     steps:
-    - name: SSH and deploy node app
+    - name: Check Out Repo
+      uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)